checkov/sast/checks_infra/base_registry.py

```diff
@@ -33,6 +37,22 @@ def extract_entity_details(self, entity: dict[str, Any]) -> tuple[str, str, dict
     def set_runner_filter(self, runner_filter: RunnerFilter) -> None:
         self.runner_filter = runner_filter
 
+    def add_external_dirs(self, external_dirs: Optional[List[str]]) -> None:
+        if external_dirs and len(external_dirs) > 0:
```

Comment:
Do we really need the added `len(external_dirs) > 0`? Isn't `if external_dirs` enough?

Reply:
right 😸